.envrc

```diff
@@ -0,0 +1,3 @@
+export MONGODB_URI=mongodb://localhost:27017/visipedia_annotation_toolkit
+direnv_layout_dir=.venv
+layout python3
```

Comment:
In the readme, it's mentioned that the code uses python 2.7. I think you either need to change the layout to `layout python2`, or update the readme: https://github.com/PassengerAI/annotation_tools#requirements-and-environments